spec/unit/daemon_spec.rb

```diff
@@ -170,10 +170,15 @@
 
       it "should log an appropriate error message and fail miserably" do
         allow(Process).to receive(:initgroups).and_raise(Errno::EPERM)
-        error = "Operation not permitted"
-        if RUBY_PLATFORM.match("solaris2") || RUBY_PLATFORM.match("aix")
-          error = "Not owner"
-        end
+        error = case RUBY_PLATFORM
+                when /solaris2/
+                  "Insufficient privileges"
+                when /aix/
+                  "Not owner"
+                else
+                  "Operation not permitted"
+                end
```

Comment:
Separately,
Do we need to continue checking the complete text, orwill it be sufficient to say behavior is correct when we see a "Permission denied" error? 

If the error string source is from an underlying OS operation, then I think we should remove it - it will make the test less brittle. 
through from the OS command?

Reply:
True